rllib/offline/json_reader.py

```diff
@@ -150,6 +150,7 @@ def _from_json(batch: str) -> SampleBatchType:
 
     if data_type == "SampleBatch":
         for k, v in data.items():
+            print("Trying to unpack {}: {}".format(k, v))#TODO
```

Comment:
Please remove this prior to merging.

Reply:
Of course, trying to catch the failing test_marwil/bc.py problem, which is related to a different compression format on the observation and will go into a different PR.